### ISSUE TITLE
Cannot get to staff/part properties dialog when something is selected

### ISSUE THREAD

**Issue by msviolafangirl** [musescore/MuseScore#18426]
### Issue type

UI bug

### Bug description

I'm a screen reader user, and I cannot access the staff/part properties dialog. I asked around, and I've been told it is possible to access it, but only if an empty area is selected, which is impossible to do with a screen reader. One solution is to add the staff/part properties dialog to the context menu more permanently. This was not an issue in MuseScore 3.

### Steps to reproduce

1. Select a note or rest in the score.
2. Press applications key or shift+F10 to open the context menu.
3. Look at the options in the context menu.

### MuseScore Version

MuseScore 4.0.2, also an issue with the latest 4.1 beta

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 10

**Comment by jmwmusic** [comment]
This is now fixed as the context menu can be opened with the application key if there is a range selection in the bar rather than clicking on an empty space.  There is also a shortcut available to customise to open this specific window.  the author of this issue is aware of these but has a problem with their GitHub account and may not be able to close this.
